hls4ml/converters/keras/core.py

```diff
@@ -120,3 +120,19 @@ def parse_batchnorm_layer(keras_layer, input_names, input_shapes, data_reader, c
         layer['n_filt']=input_shapes[0][3]
 
     return layer, [shape for shape in input_shapes[0]]
+
+@keras_handler('Permute')
```

Comment:
The organization of handlers should follow Keras's convention, and by that `Permute` layer is not a "core" layer, rather a "reshaping" layer, so it should be placed in `reshape.py` module along with other similar handlers.

Reply:
I totally agree.. I copied the TF.Keras convention from here: https://github.com/tensorflow/tensorflow/blob/v2.5.0/tensorflow/python/keras/layers/core.py#L567-L621
which defines it in core, but I'm happy to move it